.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: go
+sudo: false
+
+matrix:
+  include:
+    - go: "1.9"
+    - go: "1.10"
+      env: VET=1
+    - go: "1.11"
+      env: GO111MODULE=off
+# TODO: add go.mod so we can test this way, too
+#    - go: "1.11"
+#      env: GO111MODULE=on
+    - go: tip
+
+script:
+  - if [[ "$VET" = 1 ]]; then make; else make deps test; fi
```

Comment:
nit: want to do `then make ci` instead of `then make`?  As it is now if you add another make rule before the `ci` rule, everything will change behavior